docs/references/protocol/transactions/types/oracleset.md

```diff
@@ -0,0 +1,97 @@
+| `LastUpdateTime`   | Number    | UInt32        | Yes       | The time the data was last updated, represented in the [ripple epoch time](https://xrpl.org/basic-data-types.html#specifying-time). |
+| `AssetClass`       | String    | Blob          | Variable  | Describes the type of asset, such as "currency", "commodity", or "index". This field is a string, up to 16 ASCII hex encoded characters (0x20-0x7E). This field is required when creating a new `Oracle` ledger entry, but is optional for updates. |
+| `PriceDataSeries`  | Array     | Array         | Yes       | An array of up to 10 `PriceData` objects, each representing the price information for a token pair. More than five `PriceData` objects require two owner reserves. |
+
+
+### PriceData Fields
+
+| Field               | JSON Type | Internal Type | Required? | Description |
+|---------------------|-----------|---------------|-----------|-------------|
+| `BaseAsset`         | String    | Currency      | Yes       | The primary asset in a trading pair. Any valid identifier, such as a stock symbol, bond CUSIP, or currency code is allowed. For example, in the BTC/USD pair, BTC is the base asset; in 912810RR9/BTC, 912810RR9 is the base asset. |
+| `QuoteAsset`        | String    | Currency      | Yes       | The quote asset in a trading pair. The quote asset denotes the price of one unit of the base asset. For example, in the BTC/USD pair, BTC is the base asset; in 912810RR9/BTC, 912810RR9 is the base asset. |
+| `AssetPrice`        | Number    | UInt64        | No        | The asset price after applying the `Scale` precision level. It's not included if the last update transaction didn't include the `BaseAsset`/`QuoteAsset` pair. |
+| `Scale`             | Number    | UInt8         | No        | The scaling factor to apply to an asset price. For example, if `Scale` is 6 and original price is 0.155, then the scaled price is 155000. Valid scale ranges are 0-10. It's not included if the last update transaction didn't include the `BaseAsset`/`QuoteAsset` pair.|
+
+`PriceData` is created or updated, following these rules:
+
+- New token pairs in the transaction are added to the object.
+- Token pairs in the transaction overwrite corresponding token pairs in the object.
+- Token pairs in the transaction with a missing `AssetPrice` field delete corresponding token pairs in the object.
+- Token pairs that only appear in the object have `AssetPrice` and `Scale` removed to signify that the price is outdated.
+
+**Note:** The order of token pairs in the transaction aren't important because each token pair uniquely identifies the location of the `PriceData` object in the `PriceDataSeries`.
+
+
+## Reserve Requirements
+
+The owner reserve requirement is 1 for one to five `PriceData` objects, and 2 for six to ten `PriceData` objects.
```

Comment:
Is 1 what? 1 XRP? Probably this is more clear:

```suggestion
An Oracle object counts as one item for purposes of the [owner reserve](/docs/concepts/accounts/reserves.md#base-reserve-and-owner-reserve) if it contains one to five `PriceData` objects, and counts as two items if it contains for six to ten `PriceData` objects.
```

Also, typically I put this information in the ledger entry page, not the transaction page. Since it sounds like Oracles are a special case, it's probably also worth adding a note to the special cases section of the Reserves page.

Reply:
Moving this to the ledger entry page and adding a note on the reserves page.